nexus/types/src/external_api/views.rs

```diff
@@ -265,6 +265,22 @@ pub struct ExternalIp {
     pub kind: IpKind,
```

Comment:
Re: `views::ExternalIp`, I don't know if we want to change this to an enum (rather than rely on IpKind) to directly hand over a `views::FloatingIp` instead of needing users to (painfully) look it up again. The console works with this as-is, so as long as `ip` is flattened out then we can make this change.